docker/build_docker.sh

```diff
@@ -170,6 +170,23 @@ function docker_build {
         # del_requirements=yes
     fi
 
+    if [ -f "pyproject.toml" -a ! -f "requirements.txt" ]; then
```

Comment:
Add a message that `requirements.txt` should not be present.

Reply:
WDYM here?